package.json

```
@@ -2,15 +2,16 @@
   "name": "svelte-burger-menu",
   "version": "1.0.5",
```

This would be breaking change so its needs a major version increase
```suggestion
  "version": "2.0.0",
```

---

Done